src/protocol/inav_nrf24l01.c

```diff
@@ -85,7 +85,7 @@
 
 #ifdef EMULATOR
 #define USE_FIXED_MFGID
-#define BIND_COUNT 4.360   // 6 seconds
+#define BIND_COUNT 4      // 6 seconds
 #define FIRST_PACKET_DELAY  1
```

Comment:
this just looks weird.  was the original a typo?

Reply:
I am not quite sure. @goebish can u help understand why this is a float which is expect to be an integer.

Reply:
I suppose that was a typo, 4 is fine for EMULATOR

Reply:
no one knows why. but goebish believe this is fine.